grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -555,6 +555,16 @@ def section_object_storage(outer_panels):
                         )
                     ],
                 ),
+                panels.timeseries_ops(
+                    "Operation Retry Rate",
```

Comment:
The name is somehow confusing since only retries for read operations done by us via tokio::Retry are monitored, not inlcuding the retries done by aws sdk controlled by RetryConfig. Is it possible to also monitor aws sdk retries as well? I just came across this thread: https://github.com/awslabs/aws-sdk-rust/discussions/707

Reply:
Agree, I'm also trying to monitor all aws retry, so just mark this pr draft. Let me try some solutions.

Reply:
Seems that currently it's hard to monitor aws sdk retries, a guy from MaterializeInc is also trying to bubble up how frequently retries happen, and the member from aws-sdk provide an indirectly solution : _use a tracing subscriber to capture the tracing events that get fired from those layers_.

For us, maybe two solutions:

1.  use [tracing subscriber](https://docs.aws.amazon.com/sdk-for-rust/latest/dg/logging.html#tracing) to capture the tracing events 
2.  refactor our ObjectStoreError, make it contain various errors of s3-sdk, and replace `AWS RetryConfig` with `tokio_retry`.

For 2, it's a relatively tedious work because `SDKError` kind has many types, it is a bit difficult to include them all.  I have discussed with @zwang28 and finally we decide not do this refactoring for the time being.

Any suggestion? @hzxa21

Reply:
I also think 2 is too complicated and not the way to go. How about quickly exploring how hard it is to using tracing subscriber to only listen on retry events and report them to metrics, just like what is mentioned [here](https://github.com/awslabs/aws-sdk-rust/discussions/707#discussioncomment-5834438)?

Reply:
+1